.eslintrc.js

```diff
@@ -96,6 +96,18 @@ module.exports = {
     // Rules for prettier.
     "prettier/prettier": errorIfStrict, // Makes Prettier issues warnings rather than errors.
     "sort-destructure-keys/sort-destructure-keys": errorIfStrict,
+    "check-file/filename-naming-convention": [
+      "warn",
+      {
+        "**/*.graphql": "KEBAB_CASE",
+        "cypress/integration/**/*.ts": "SNAKE_CASE",
+        "scripts/**/*.{js,ts}": "KEBAB_CASE",
+        "src/**/*.{js,ts}": "CAMEL_CASE",
```

Comment:
You can use `!(something)` syntax to not match on values that can be hooks.

e.g.
`src/**/!(use)*.ts` Will match all ts files that do not have the use keyword aka are not hooks.

Checkout for an easy way to test out glob match rules.
https://www.digitalocean.com/community/tools/glob?comments=true&glob=src/**/!(use)*.tsx&matches=false&tests=src/hooks/useTestFile.ts&tests=src/components/SomeRandomComponent.tsx